inst/report/scripts/run_molevolvr_pipeline.R

```diff
@@ -0,0 +1,1296 @@
+
+  for (i in 1:nrow(blast_clustnames)) {
+    blast_clustnames$ClusterID[i] <- str_c(blast_clustnames$ClusterID[i])
+  }
+
+  first_prot <- as.data.frame(word(clust_out$AccNum), word(clust_out$ClusterID))
+
+  ## write the new file as a TSV file
+  newarg <- gsub(".bclust.L[0-9][0-9]S[0-9][0-9].tsv", "", clust)
+  # accnum + clusterID
+  write_tsv(new_clust, file = paste0(newarg, ".clustIDs"), append = F)
+  # first protein from every cluster
+  write_tsv(first_prot, file = paste0(newarg, ".clust_reps"),
+            col_names = F, append = F)
+  # cleaned up blast file + clusterIDs
+  write_tsv(blast_clustnames, file = paste0(newarg, ".cln.clust.tsv"),
+            col_names = T, append = F)
+}
+
+# Function to run InterProScan
+run_interproscan <- function(query_file, prefix, outdir) {
+
+  # Start InterProScan run
+  cat("\n######################\n")
+  cat("BEGIN INTERPROSCAN RUN\n")
+  cat("Input file:", query_file, "\n")
+  cat("######################\n")
+
+  # Output file path
+  outfile <- file.path(outdir, paste0(prefix, ".iprscan"))
+
+  # Process the input query file with InterProScan
+  cat("Now processing", query_file, "\n")
+
+  # Run InterProScan command
+  # Construct the command
+  command <- paste(
+    "~/iprdir/interproscan-5.70-102.0/interproscan.sh -i",
+    shQuote(query_file),
+    "-b", shQuote(outfile),
+    "-f TSV --cpu", Sys.getenv("INTERPROSCAN_CPUS", "4"),
+    "--appl Pfam,MobiDBlite,Phobius,Coils,SignalP_GRAM_POSITIVE,",
+    "SignalP_GRAM_NEGATIVE,Hamap,Gene3D,SignalP_EUK"
+  )
+
+  # Run the command
+  system(command)
+
+  cat("##################\n")
+  cat("END OF IPRSCAN RUN\n")
+  cat("##################\n")
+}
+
+ipr2lin <- function(ipr, acc2info, prefix) {
+  # read in iprscan results
+  # duplicate rows in iprscan file
+  ipr_in <- read_tsv(ipr, col_names = TRUE) %>%
+    mutate(DB.ID = gsub("G3DSA:", "", DB.ID))
+
+  acc2info_out <- fread(input = acc2info, sep = "\t", header = T, fill = T) %>%
+    mutate(FullAccNum = gsub("\\|", "", FullAccNum)) %>%
+    mutate(FullAccNum = gsub(".*[a-z]", "", FullAccNum))
+
+  # merge ipr file with acc2info file
+  ipr_in <- ipr_in %>%
+    # remove version number and any other suffices
+    mutate(AccNum.noV = gsub("\\.[0-9].*", "", AccNum))
+
+  ipr_tax <- left_join(ipr_in, acc2info_out, by = "AccNum")
+
+  # read in lineage map
+  lineage_map <- fread("~/awasyn/new_trial/lineage_lookup.txt",
+                       header = T, fill = T)
+
+  # merge ipr+info w/ lineage
+  # both tables have a species column, but only
+  # the lineage_map (y) species column is kept
+  ipr_tax <- ipr_tax %>%
+    mutate(TaxID = as.numeric(TaxID))
+
+  ipr_lin <- left_join(ipr_tax, lineage_map, by = "TaxID") |>
+    mutate(Species = Species.y) %>%
+    select(-Species.x, -Species.y)
+
+  # add lookup table to iprscan file
+  lookup_tbl <- fread(input = "~/awasyn/new_trial/cln_lookup_tbl.tsv",
+                      sep = "\t", header = T, fill = T) %>%
+    distinct()
```

Comment:
Sorry for the repetition! This is another invocation that should IMHO be replaced with `system.file()`.